### ISSUE TITLE
Heatmap: click event doesn't work properly when tooltip is disabled

### ISSUE THREAD

**Issue by DavidSvetlik** [apexcharts/apexcharts.js#1837]
# Bug report

## Explanation

Problem is that when using heatmap, I want to disable tooltip via `options.tooltip.enabled = false;`. However, mouse click event (and maybe some others) stop working correctly as although click event still triggers, `dataPointIndex` and `seriesIndex` have `-1` value.

I am using React version of apexcharts but I assume this bug is tied to core library.

Thank you,
David Svetlik

**Comment by seth-imbellus** [comment]
I'm dealing with the same issue but within the Vue wrapper and on a line chart. If I disable the tooltips the click events stop working.

**Comment by shijiezhou1** [comment]
```
 tooltip: {
          enable: false,
          x: {
            formatter: function(val) {
              return "";
            },
          },
          y: {
            formatter: function(val) {
              return "";
            },
          }
        },```

I tried to disable the tooltips but it seems not to work.

**Comment by pee0803** [comment]
is this issue fixed? Cause im facing this issue now with v1.4.0
